src/js/components/molecules/flyout-section/flyout-section.ts

```diff
@@ -0,0 +1,129 @@
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
+import { NavigationItem  } from '@ultimaker/ultimaker.com-model-definitions/dist/molecules/navigation-item/NavigationItem';
+
+@Component({
+    name: 'flyout-section',
+    template: require('./flyout-section.html'),
+})
+
+export default class Flyout extends Vue  {
+    @Prop({ type: String, required: true }) label!: string;
+    @Prop({ type: Array, required: false }) items?: NavigationItem[];
+
+    @Prop({ type: Object, required: false }) bottomLink?: object;
+    @Prop({ type: Boolean, required: false }) isCompact?: boolean;
+    @Prop({ type: Number, required: false, default: 3 }) maxVisible?: number;
+    @Prop({ type: Number, required: false, default: 6 }) maxItemsCol?: number;
+
+    @Prop({ type: String, required: false, default: () => `flyout_title_${ ~~(Math.random() * 10000) }` }) itemId?: string;
+
+    isExpanded: boolean =  false;
+    focusIndex: null | number = null;
+    columnClassDouble: string = 'flyout__section--2-columns';
+    columnClassTriple: string = 'flyout__section--3-columns';
+
+    get hasItems(): NavigationItem[] | undefined {
+        return this.items;
+    }
+
+    get focusableItems() {
+        return Array.isArray(this.$refs.focusable) ? this.$refs.focusable : [this.$refs.focusable];
+    }
+
+    get moreThanMaxItems(): boolean {
+        // @ts-ignore
```

Comment:
Can you explain why the linting rules are ignored, or if it needs to be fixed a todo for now?